### PR TITLE
docs: fix and sync documents

### PR DESCRIPTION
**Pull request by misitebao**
- [x] Standardize JavaScript and TypeScript name writing
- [x] Sync translated documents

Preview: https://misitebao-wails-git-feature-documents-misitebao.vercel.app

### CODE REVIEW

File: website/i18n/ja/docusaurus-plugin-content-docs/current/guides/templates.mdx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# テンプレート
```
Comment (leaanthony) [inline website/i18n/ja/docusaurus-plugin-content-docs/current/guides/templates.mdx:1]: Are we reverting this from Japanese?

File: website/i18n/ja/docusaurus-plugin-content-docs/current/howdoesitwork.mdx [hunk 2]
```diff
@@ -4,7 +4,7 @@ sidebar_position: 20
 
 # どうやって動いているの?
 
-Wailsは、webkitフロントエンドを備えた、何の変哲もないGoアプリです。 アプリ全体のうちGoの部分は、アプリのコードと、ウィンドウ制御などの便利な機能を提供するランタイムライブラリで構成されています。 フロントエンドはwebkitウィンドウであり、フロンドエンドアセットをウィンドウ上に表示します。 フロントエンドからも、Javascriptでランタイムライブラリを呼び出すことができます。 そして最終的に、Goのメソッドはフロントエンドにバインドされ、ローカルのJavascriptメソッドであるかのように、フロントエンドから呼び出すことができます。
```
Comment (leaanthony) [inline website/i18n/ja/docusaurus-plugin-content-docs/current/howdoesitwork.mdx:7]: Are we reverting this from Japanese?

**Review by leaanthony — APPROVED** [review]
Noticed there were quite a few pages where we are going from Japanese back to English. Was this intentional?

File: website/i18n/ja/docusaurus-plugin-content-docs/current/howdoesitwork.mdx (hunk 2 above)
Comment (misitebao) [reply website/i18n/ja/docusaurus-plugin-content-docs/current/howdoesitwork.mdx:7]: I removed those auto-added machine translations and copied English translations when I added Japanese, I don't know how those were generated, but you were the translator and approver of those characters, so I removed them all and let Japanese translators re-translate to ensure translation accuracy.

**Comment by leaanthony** [comment]
🙏

**Pull request merged**